src/contexts/SessionEra/index.tsx

```diff
@@ -72,19 +71,17 @@ export const SessionEraProvider = ({
   const getEraTimeLeft = () => {
     const eraBlocksLeft =
       sessionEraRef.current.eraLength - sessionEraRef.current.eraProgress;
-    const eraTimeLeftSeconds = eraBlocksLeft * (expectedBlockTime * 0.001);
 
-    const unixTime = getUnixTime(new Date());
-    const eventTime = unixTime + eraTimeLeftSeconds;
-    const diffTime = eventTime - unixTime;
-    return diffTime;
+    const eraTimeLeftSeconds = eraBlocksLeft * (expectedBlockTime * 0.001);
+    return eraTimeLeftSeconds;
   };
 
   return (
     <SessionEraContext.Provider
       value={{
         getEraTimeLeft,
         sessionEra,
+        eraTimeLeft: getEraTimeLeft(),
```

Comment:
The `getEraTimeLeft` function does not really introduce any tangible performance gains (which is what I believe was a reason for returning a function instead of simply the actual value) for 2 reasons:
* the `getEraTimeLeft` function is not memoized, so it's recreated every single time that it's dependencies values are changed, therefore any potential performance gain on limited rerenders is lost anyway,
* dependencies aside, the calculations inside are minuscule, so there's also no tangible performance gain by limiting them.

Hence I just added the actual value itself returned from the hook (and left the function for backwards compatibility).